layers/+spacemacs/spacemacs-modeline/packages.el

```diff
@@ -133,17 +133,18 @@
                           (if (yes-or-no-p
                                (format
                                 (concat "Do you want to update to the newest "
-                                        "version %s ?") spacemacs-new-version))
+                                        "version %s ?")
+                                spacemacs-new-version))
                               (progn
                                 (spacemacs/switch-to-version
                                  spacemacs-new-version))
                             (message "Update aborted."))))
                       map)))
       (spaceline-define-segment new-version
-        (when spacemacs-new-version
-          (spacemacs-powerline-new-version
-           (spacemacs/get-new-version-lighter-face
-            spacemacs-version spacemacs-new-version))))
+                                (when spacemacs-new-version
```

Comment:
How about

```
      (spaceline-define-segment
       new-version
       (when spacemacs-new-version
         (spacemacs-powerline-new-version
          (spacemacs/get-new-version-lighter-face
           spacemacs-version spacemacs-new-version))))
```

Reply:
Your right, that's better, less empty space to the left.